smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/TypeScriptDependency.java

```diff
@@ -47,7 +47,7 @@
 
     AWS_SDK_STREAM_COLLECTOR_NODE("dependencies", "@aws-sdk/stream-collector-node", "^1.0.0-alpha.1", true),
     AWS_SDK_STREAM_COLLECTOR_BROWSER("dependencies", "@aws-sdk/stream-collector-browser", "^1.0.0-alpha.1", true),
-    AWS_SDK_STREAM_COLLECTOR_RN("dependencies", "@aws-sdk/stream-collector-rn", "^1.0.0-alpha.0", true),
+    AWS_SDK_STREAM_COLLECTOR_RN("dependencies", "@aws-sdk/stream-collector-native", "^1.0.0-alpha.0", true),
```

Comment:
Why is this necessary?

Reply:
Since we're preferring `.native` vs `.rn` extensions we should also move from `-rn` to `-native` package names.

Similar to `.browser` extension and `-browser` package name

Reply:
Relevant commit for updating package names in codegen https://github.com/aws/aws-sdk-js-v3/pull/1025/commits/bb1e16bf63ba2a4446d4f962952b0c8e71611c14

Reply:
I'm not apposing to it, but I don't think it's necessary either.🙂